lib/git_op.rb

```diff
@@ -17,19 +17,24 @@ def initialize(git_dir, pr)
     @repo_external = ExternalRepoGit.new(pr)
   end
 
-  def ck_or_clone_git(repo)
+  def ck_or_clone_git(repo, https)
     return if File.directory?(git_dir)
     FileUtils.mkdir_p(git_dir)
     Dir.chdir git_dir
-    puts `git clone git@github.com:#{repo}.git`
+    repo_url = if https
```

Comment:
Here is better to have
```if_this_is_a_true_value ? then_the_result_is_nthis : else_it_is_this```
https://stackoverflow.com/questions/4252936/how-do-i-use-the-conditional-operator-in-ruby

Reply:
Ok, since the ternary operator won't fit in to a line with less than 80 characters, and I can't split the line since ternary operators should not use more than one line, that forces me to setup the variable twice:

* One time to store "git@github.com:" or "https://github.com/"
* Another time to add the project name, repo name and '.git'

Let me give it a try and will amend the commit.

Reply:
Fixed